include/boost/geometry/extensions/gis/io/wkb/detail/writer.hpp

```diff
@@ -178,14 +180,16 @@ namespace detail { namespace wkb
 
             interior_rings_type interior_rings = boost::geometry::interior_rings(polygon);
```

Comment:
Good catch!
I just used the correct `..._return_type` for them to not modify too much.